Align with template and D3 .1 (#147)
APTITUDE-Consortium/aptitude-eudi-wallet-specs: docs/rulebook/dtc-rulebook.md

Reviewer: ```suggestion
```

Reviewer: ```suggestion
```

Reviewer: delete entire clause as it has been moved to clause 2, all lines 101 - 121

Reviewer: move requirements in lines 129 to 137 to clause 2

Reviewer: ```suggestion
### 3.1.2 General PhotoID data elements
```

Reviewer: ```suggestion
| **Identifier** | **additional description** | **Presence in APTITUDE DTC** |
| --- | --- | --- |
| ``family_name`` | no further information | M |
| ``given_name`` | no further information | M  |
```

Reviewer: ```suggestion
The data elements are specified in Table 1 in clause 3.1.2.

— The "Identifier" column is the reference of the data element specified in [ISO/IEC 23220-4] and [ISO/IEC 23220-2].

— The “additional description” column gives further information on the usage of the respective data element in the context of APTITUDE DTC. The description in [ISO/IEC 23220-4] applies.

— The "Presence" column indicates whether the presence of the element on an APTITUDE DTC is mandatory (M), optional (O) or conditional (C). A mandatory data elemement SHALL be present in an APTITUDE DTC whereas an optional data element MAY be present. If a data element is conditional the respective condition is given in the specification. If the condition is met the data element SHALL be present.
```

Reviewer: ```suggestion
```

Reviewer: ```suggestion
```

Reviewer: ```suggestion
```

Reviewer: ```suggestion
```

Reviewer: ```suggestion
```

Reviewer: ```suggestion
```

Reviewer: ```suggestion
```

Reviewer: ```suggestion
```

Reviewer: ```suggestion
### 2.1 Chapter overview and general requirements
```

Reviewer: ```suggestion
```

> # APTITUDE - *Digital Travel Credential (DTC) rulebook*

Reviewer: ```suggestion
# Attestation Rulebook for attestations of type  *APTITUDE DTC*
```

| Index | Requirement specification |
| --- | --- |
| XX_XX | According to ICAO’s DTC-VC data model, the APTITUDE DTC SHALL contain DG1, DG2, SOD as from the physical eMRTD passport.|
| XX_XX | According to ICAO's DTC-VC data model, the APTITUDE DTC MAY contain fields like: dtcSecurityInfo, DTCIdentifier, DTCDOE, and a signature structure for validation in case of eMRTD-PC bound    |
| XX_XX | The APTITUDE DTC SHALL be encapsulated as a Verifiable Credential (VC), ensuring compatibility with the EUDI Wallet data formats (e.g., SD-JWT or mDoc acc. ISO 23220-4).                      |
| XX_XX | APTITUDE DTC MAY contain additional attributes beyond the derived eMRTD dataset                                                                                                                |
| XX_XX | The data model SHALL support Selective Disclosure, allowing the traveller to share only the strictly necessary attributes (e.g., only DG2 for biometric match) with Relying Parties.           |

> ### 2.2 Requirements

Reviewer: ```suggestion
### 2.2 Technical requirements
```

> This chapter defines the credential format and encoding requirements for the APTITUDE Digital Travel Credential (DTC). The APTITUDE DTC SHALL use exactly one credential format: the ISO/IEC 23220-4 PhotoID profile, encoded as ISO/IEC 18013-5 mdoc-cbor.

Reviewer: ```suggestion
This chapter defines the credential format and encoding requirements for the APTITUDE Digital Travel Credential (DTC).
```

| DTC_DM_05 | APTITUDE DTC MAY contain additional attributes beyond the derived eMRTD dataset |
| DTC_DM_06 | The data model SHALL support Selective Disclosure, allowing the traveller to share only the strictly necessary attributes (e.g., only DG2 for biometric match) with Relying Parties. |

## 3 Attestation Encoding

### 3.1 ISO/IEC 18013-5-compliant encoding

> This chapter defines the credential format and encoding requirements for the APTITUDE Digital Travel Credential (DTC). The APTITUDE DTC SHALL use exactly one credential format: the ISO/IEC 23220-4 PhotoID profile, encoded as ISO/IEC 18013-5 mdoc-cbor.

Reviewer: ```suggestion
```

> This chapter defines the credential format and encoding requirements for the APTITUDE Digital Travel Credential (DTC). The APTITUDE DTC SHALL use exactly one credential format: the ISO/IEC 23220-4 PhotoID profile, encoded as ISO/IEC 18013-5 mdoc-cbor.

Reviewer: ```suggestion
```

| DTC_DM_05 | APTITUDE DTC MAY contain additional attributes beyond the derived eMRTD dataset |
| DTC_DM_06 | The data model SHALL support Selective Disclosure, allowing the traveller to share only the strictly necessary attributes (e.g., only DG2 for biometric match) with Relying Parties. |

## 3 Attestation Encoding

### 3.1 ISO/IEC 18013-5-compliant encoding

#### 3.1.1 APTITUDE DTC document type and namespaces

The objects ``docType`` and ``namespace`` are used to encapsulate the document type and the space in which the data elements are defined.

The document type for the **APTITUDE DTC** SHALL be as specified in clause C.2.1 of [ISO/IEC 23220-4].

*Note* The document type is ``org.iso.23220.photoid.1``.

The namespace of APTITUDE DTC for **general PhotoID data elements** defined in clause 3.1.2 SHALL be as specified in clause C.2.1 of [ISO/IEC 23220-4] and 6.3.1.1 of [ISO/IEC 23220-2].
The namespace for **specific PhotoID data elements** defined in clause 3.1.3 SHALL be as specified in clause C.2.1 of [ISO/IEC 23220-4]. 
### 3.1.2 General PhotoID data elements

> The general PhotoID data elements of APTITUDE DTC SHALL be as defined in Table 1 and belong to the namespace given in 3.1.1.

Reviewer: ```suggestion
The general PhotoID data elements of APTITUDE DTC SHALL be as defined in Table 1 and belong to the namespace given in 3.1.1.

— The "Identifier" column is the reference of the data element specified in [ISO/IEC 23220-4].

— The “additional description” column gives further information on the usage of the respective data element in the context APTITUDE DTC.

— The "Presence" column indicates whether the presence of the element on an EU-mVRC is mandatory (M), optional (O) or conditional (C). A mandatory data elemement SHALL be present in an APTITUTED DTC whereas an optional data element MAY be present. If a data element is conditional the respective condition is given in the specification. If the condition is met the data element SHALL be present.

```

| DTC_DM_05 | APTITUDE DTC MAY contain additional attributes beyond the derived eMRTD dataset |
| DTC_DM_06 | The data model SHALL support Selective Disclosure, allowing the traveller to share only the strictly necessary attributes (e.g., only DG2 for biometric match) with Relying Parties. |

## 3 Attestation Encoding

### 3.1 ISO/IEC 18013-5-compliant encoding

#### 3.1.1 APTITUDE DTC document type and namespaces

The objects ``docType`` and ``namespace`` are used to encapsulate the document type and the space in which the data elements are defined.

The document type for the **APTITUDE DTC** SHALL be as specified in clause C.2.1 of [ISO/IEC 23220-4].

> The namespace for **general PhotoID data elements** defined in clause 3.1.2 SHALL be as specified in clause C.2.1 of [ISO/IEC 23220-4] and 6.3.1.1 of [ISO/IEC 23220-2].

Reviewer: ```suggestion
The namespace of APTITUDE DTC for **general PhotoID data elements** defined in clause 3.1.2 SHALL be as specified in clause C.2.1 of [ISO/IEC 23220-4] and 6.3.1.1 of [ISO/IEC 23220-2].

*Note* The namespace identifier is ``org.iso.23220.1``.
```

> 

Reviewer: ```suggestion
The namespace for **specific PhotoID data elements** defined in clause 3.1.3 SHALL be as specified in clause C.2.1 of [ISO/IEC 23220-4]. 

*Note:* The namespace identifier is ``org.iso.23220.photoID.1``.

The namespace for **ICAO PhotoID data elements** defined in clause 3.1.4 SHALL be as specified in clause C.2.1 of [ISO/IEC 23220-4]. 

*Note:* The namespace identifier is ``org.iso.23220.datagroups.1``.

Member States MAY add additional namespaces under their responsibility. The namespace for Member State specific data elements SHALL be as specified in clauses C.2.1 of [ISO/IEC 23220-4]

 *Example* The domestic namespace of the European Union is ``org.iso.23220.photoID.EU.1``, with number 1 indicating the version of the domestic namespace. 
```

### 3.1.2 General PhotoID data elements

> For proximity presentation and wallet-native use, APTITUDE DTC SHALL use ISO/IEC 18013-5 mdoc-cbor with the PhotoID payload defined by ISO/IEC 23220-4 Annex C.

Reviewer: ```suggestion
The general PhotoID data elements of APTITUDE DTC SHALL be as defined in Table 1 and belong to the namespace given in 3.1.1.
```

*Note* The namespace identifier is ``org.iso.23220.photoid.1``.

Member States MAY add additional namespaces under their responsibility. The namespace for Member State specific data elements** SHALL be as specified in clauses C.1.2 of [ISO/IEC 23220-4].

*Note* The domestic namespace of the European Union is ``org.iso.23220.photoid.EU.1`` with number 1 indicating the version of this namespace.

The next section define general and specific data elements of the PhotoID.
These elements are given in tables of three columns:

- The "Identifier" column is the reference of the data element specified in [ISO/IEC 23220-4].

- The “additional description” column gives further information on the usage of the respective data element in the context APTITUDE DTC.

- The "Presence" column indicates whether the presence of the element on an EU-mVRC is mandatory (M), optional (O) or conditional (C). A mandatory data elemement SHALL be present in an APTITUTED DTC whereas an optional data element MAY be present. If a data element is conditional the respective condition is given in the specification. If the condition is met the data element SHALL be present.

#### 3.1.2 General PhotoID data elements

> The general PhotoID data elements of APTITUDE DTC SHALL be as defined in Table 1 and belong to the namespace given in 3.1.1.

Reviewer: ```suggestion
The general PhotoID data elements of APTITUDE DTC SHALL be as defined in Table 1 and belong to the namespace given in 3.1.1. The data element definitions given in clause C.2.1 and Table C.1 in [ISO/IEC 23220-4] apply if not stated otherwise. 
```

###### Table 1 — general PhotoID data elements

| **Identifier** | **additional description** | **Presence in APTITUDE DTC** |
| --- | --- | --- |
| ``family_name`` | no further information | M |
| ``given_name`` | no further information | M  |

#### 3.1.3 Specific PhotoID data elements

> 

Reviewer: ```suggestion
The specific PhotoID data elements of APTITUDE DTC SHALL be as defined in Table 2 and belong to the namespace given in 3.1.1. The data element definitions given in clause C.2.1 and Table C.2 in [ISO/IEC 23220-4] apply if not stated otherwise.
```

###### Table 2 — specific PhotoID data elements

| **Identifier** | **additional description** | **Presence in APTITUDE DTC** |
| --- | --- | --- |
| ``person_id`` | no further information | M |

> | ``birth_country`` | no further information | M |

Reviewer: ```suggestion
| ``birth_country`` | no further information | O |
| ``birth_state`` | no further information | O |
| ``birth_city`` | no further information | O |
| ``administrative_number`` | no further information | O |
| ``resident_street`` | no further information | O |
| ``resident_house_number`` | no further information | O |
| ``travel_document_type`` | no further information | O |
| ``travel_document_number`` | no further information | O |
| ``resident_state`` | no further information | O |
| ``travel_document_mrz`` | no further information | O |
```

#### 3.1.4 ICAO PhotoID data elements 

> 

Reviewer: ```suggestion
The ICAO PhotoID data elements of APTITUDE DTC SHALL be as defined in Table 3 and belong to the namespace given in 3.1.1. The data element definitions given in clause C.2.1 and Table C.3 in [ISO/IEC 23220-4] apply if not stated otherwise.
```

###### Table 3 — ICAO PhotoID data elements

| **Identifier** | **EU additional description** | **Presence in EU-mVRC** |
| --- | --- | --- |
| ``version`` | no further information | M |
| ``dg1`` | no further information | M |

## 4 Attestation Usage

> 

Reviewer: ```suggestion
t.b.d.
```

## 5 Trust Anchors

The APTITUDE DTC is derived from the physical eMRTD LDS data groups and signed by the national issuing authority. The PhotoID credential SHALL carry the same derived content as the DTC data model, ensuring the credential remains linked to the physical document and the wallet secure component.

## 6 Revocation

Revocation of the APTITUDE DTC SHALL be implemented according to ISO 18013-5 2nd edition, i.e. MSO revocation information..

## 7 Compliance

> 

Reviewer: ```suggestion
If compliance to ICAO DTC-VC is required, a reader may after succsessfull processing the device response and the verification prodecure encapsule the data in the structure given in clause 7.1.  
```

| DTC_AE_07 | APTITUDE DTC SHALL preserve the cryptographic binding between the virtual credential and the Wallet Secure Component across issuance, storage, presentation, and verification. |
| DTC_AE_08 | APTITUDE DTC SHALL support selective disclosure and minimisation as a layer on top of the single PhotoID credential format, not by introducing a second credential format. |

> The APTITUDE DTC SHALL be encoded using the ISO/IEC 23220-4 PhotoID profile and carried as an ISO/IEC 18013-5 mdoc-cbor payload with document type `org.iso.23220.photoID.1`.

Reviewer: ```suggestion
#### 3.1.1 APTITUDE DTC document type and namespaces

The objects ``docType`` and ``namespace`` are used to encapsulate the document type and the space in which the data elements are defined.

The document type for the **APTITUDE DTC** SHALL be as specified in clause C.2.1 of [ISO/IEC 23220-4].
```

> * the DTC is derived from physical eMRTD LDS data groups,

Reviewer: ```suggestion
```

> * the DTC is derived from physical eMRTD LDS data groups,

Reviewer: ```suggestion
The namespace for **specific PhotoID data elements** defined in clause 3.1.3 SHALL be as specified in clause C.2.1 of [ISO/IEC 23220-4].

The namespace for **ICAO data elements** defined in clause 3.1.4 SHALL be as specified in clause C.2.1 of [ISO/IEC 23220-4].

Member States MAY add additional namespaces under their responsibility. The namespace for **Member State specific PhotoID data elements** SHALL be as specified in clauses C.2.1 of [ISO/IEC 23220-4].
```

> * the PhotoID format supports the border-control use case and wallet presentation flows.

Reviewer: ```suggestion
```

> ### 3.1.1 Supported encoding and semantics

Reviewer: ```suggestion
```

> ### 3.1.1 Supported encoding and semantics

Reviewer: ```suggestion
```

> The mdoc SHALL include the document type `org.iso.23220.photoID.1` and SHALL contain the APTITUDE DTC payload as attestation content.

Reviewer: ```suggestion
```

### 3.1.3 Specific PhotoID data elements

| Data Identifier | Attribute identifier | Encoding format | Namespace |
| --- |----------------------| --- | --- |
| family_name | family_name          | tstr | org.iso.23220.photoID.1 |
| given_name | given_name           | tstr | org.iso.23220.photoID.1 |
| date_of_birth | date_of_birth        | tdate | org.iso.23220.photoID.1 |
| document_number | document_number      | tstr | org.iso.23220.photoID.1 |
| nationality | nationality          | tstr | org.iso.23220.photoID.1 |
| photo | face_image           | bstr | org.iso.23220.photoID.1 |
| sod | dg1                  | bstr | org.iso.23220.datagroups.1 |
| sod | dg2                  | bstr | org.iso.23220.datagroups.1 |
| sod | dg14                 | bstr | org.iso.23220.datagroups.1 |
| sod | sod                  | bstr | org.iso.23220.datagroups.1 |

> ### 3.1.4 Relationship to APTITUDE architecture

Reviewer: ```suggestion
### 3.1.4 ICAO PhotoID data elements 
```

> The APTITUDE DTC is derived from the physical eMRTD LDS data groups and signed by the national issuing authority. The PhotoID credential SHALL carry the same derived content as the DTC data model, ensuring the credential remains linked to the physical document and the wallet secure component.

Reviewer: this should be moved to new claus 5 on trust anchors

> The APTITUDE DTC SHALL approximate an ICAO DTC Type 2 credential, by providing a PhotoID mdoc as DTC-VC and allowing for the eMRTD to serve as DTC-PC, unless the mobile device's Wallet Secure Component supports strong proof of possession.

Reviewer: ```suggestion
###### Table 3 — ICAO PhotoID data elements

| **Identifier** | **EU additional description** | **Presence in EU-mVRC** |
| --- | --- | --- |
| ``version`` | no further information | M |
| ``dg1`` | no further information | M |
```

> ### 3.2 ICAO based encoding

Reviewer: move to new chapter 7 compliance

> ### 3.1.3 Attribute namespace and identifiers

Reviewer: ```suggestion
### 3.1.3 Specific PhotoID data elements
```

> The APTITUDE DTC credential format profile is designed around four namespaces:

Reviewer: ```suggestion
```

> * `org.iso.23220.photoID.1` contains the data elements for the photoID

Reviewer: ```suggestion
```

> * `org.iso.23220.datagroups.1` contains the datagroups defined by ICAO 9303

Reviewer: ```suggestion
```

> * `org.iso.23220.1` contains information for holder binding, will not be used for APPTITUDE DTC

Reviewer: ```suggestion
```

> * A domestic or profile-specific namespace for any rulebook-specific attributes that are not part of the EU-wide core DTC profile.

Reviewer: ```suggestion
```
